src/app.ts

```diff
@@ -11,4 +11,16 @@ app.use('/api/users', usersRouter);
 app.use(imagesRouter);
 app.use('/api/albums', albumsRouter);
 
+const errorHandler: ErrorRequestHandler = (error, req, res, next) => {
+  const statusCode = error.statusCode || 500;
+  const data = error.data;
+  const message = error.message;
+  res.status(statusCode).json({
+    message: message,
+    error: data,
+  });
+};
```

Comment:
I'd anticipate seeing the handler in a separate file.

Reply:
Moved to separate file